test/danetest.c

```diff
@@ -389,111 +360,71 @@ static int test_tlsafile(SSL_CTX *ctx, const char *base_name,
         SSL_set_verify_result(ssl, err);
         SSL_free(ssl);
 
-        if (ok < 0) {
+        if (!TEST_int_eq(err, want)) {
+            if (want == X509_V_OK)
+                TEST_info("Verification failure in test %d: %d=%s",
+                          testno, err, X509_verify_cert_error_string(err));
+            else
+                TEST_info("Unexpected error in test %d", testno);
             ret = 0;
-            fprintf(stderr, "verify_chain internal error in %s test %d\n",
-                    path, testno);
-            print_errors();
             continue;
         }
-        if (err != want || (want == 0 && !ok)) {
+        if (!TEST_false(want == 0 && ok == 0)) {
+            TEST_error("Verification failure in test %d: ok=0", testno);
```

Comment:
So here, you're making it a `TEST_error`, but just a few lines up it's a `TEST_info`.  Please make them the same.